vnsiclient.c

```diff
@@ -280,8 +280,6 @@ bool cVNSIClient::EpgChange()
     }
     it->second.attempts++;
 
-    INFOLOG("Trigger EPG update for channel %s, id: %d", channel->Name(), channelId);
```

Comment:
I don't consider this useless, otherwise I wouldn't added it. Using vdr logging levels or introducing vnsi logging levels would be better.